subprojects/build-cache/src/main/resources/db/migration/V001__init_schema.sql

```diff
@@ -1,11 +1,11 @@
-CREATE SCHEMA filestore;
+CREATE SCHEMA IF NOT EXISTS filestore;
 
-CREATE TABLE filestore.catalog
+CREATE TABLE IF NOT EXISTS filestore.catalog
 (
     id            BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
     entry_key     VARCHAR(32)         NOT NULL,
     entry_size    BIGINT              NOT NULL,
     entry_content BINARY LARGE OBJECT NOT NULL
 );
 
-CREATE INDEX idx_catalog_entry_key ON filestore.catalog (entry_key);
+CREATE INDEX IF NOT EXISTS idx_catalog_entry_key ON filestore.catalog (entry_key);
```

Comment:
Do we need this index now? Won't the primary key get indexed automatically?

Reply:
Check next commit, should fix that